apluslms_roman/cli.py

```diff
@@ -305,13 +326,23 @@ def build_action(context):
     config = get_config(context)
     engine = get_engine(context)
 
+    builder = engine.create_builder(config)
+
+    # TODO: option to print more?
+    if (context.parser.parse_args().steps):
+        steps = builder.get_steps()
+        num_len = len(str(len(steps)))
+        name_len = len(max(steps, key=lambda s: len(s.name or "")).name)
+        for step in steps:
+            step_str = "{:%dd}. {:%ds} {}" % (num_len, name_len)
```

Comment:
No need recalculate for every iteration (no changes to data per item in for loop). So, can be moved before for.